builtin/verify-tag.c

```diff
@@ -51,8 +56,20 @@ int cmd_verify_tag(int argc, const char **argv, const char *prefix)
 		const char *name = argv[i++];
 		if (get_sha1(name, sha1))
 			had_error = !!error("tag '%s' not found.", name);
-		else if (gpg_verify_tag(sha1, name, flags))
-			had_error = 1;
+		else {
+			if (fmt_pretty) {
+				verify_ref_format(fmt_pretty);
```

Comment:
By *"merge this code"*, you mean moving it to [`tag.c#gpg_verify_tag`](https://github.com/lukpueh/git/blob/master/tag.c#L33-L60) where `git tag -v` and `git verify-tag` call into?

I think if we do move, which we should, we still want to verify the format string only once for all tags. I.e. I'm in favor for one `verify_ref_format` in `builtin/tag.c` and one in `builtin/verify-tag.c`.